packages/fx-core/src/plugins/resource/frontend/configs.ts

```diff
@@ -79,21 +74,38 @@ export class FrontendConfig {
     }
 
     return new FrontendConfig(
-      appName,
       subscriptionId,
       resourceGroupName,
       location,
-      resourceNameSuffix,
       storageName,
       credentials
     );
   }
 
+  public syncToPluginContext(ctx: PluginContext): void {
+    Object.entries(this)
+      .filter((kv) => FrontendConfig.persistentConfigList.find((x: string) => x === kv[0]))
```

Comment:
Nit: could be `FrontendConfig.persistentConfigList.find(kv[0])`

Reply:
seems `find` only consume a callback, use includes instead.